python/pfs/drp/stella/createDetGeom.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import argparse
+
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        super(CreateDetGeomTask, self).__init__(*args, **kwargs)
```

Comment:
LSST discourages the use of `super` unless it's really, really necessary: see [the Developer Guide](https://developer.lsst.io/coding/python_style_guide.html#super-should-not-be-used-unless-the-author-really-understands-the-implications-e-g-in-a-well-understood-multiple-inheritance-hierarchy). I don't know if PFS has formally adopted the LSST standards, but probably following them would be a good idea.